### PR TITLE
mkdir on data import if not existing

### LINKED ISSUE #501: data import should mkdir in the container if it doesn't exit
I'm on the fence. Would be ++ convenient but seems very non-unixy

### PR DESCRIPTION
**Pull request by zramsay**
closes #501 
requires https://github.com/eris-ltd/eris-by-example/pull/3 (a test) to pass

### CODE REVIEW

File: data/operate.go [hunk 1]
```diff
@@ -65,19 +86,16 @@ func ImportData(do *definitions.Do) error {
...
+		// and eris images have the `eris` user by default
+		if err := runData(containerName, []string{"chown", "--recursive", "eris", do.Destination}); err != nil {
+			return err
 		}
+
 	} else {
-		log.WithField("name", do.Name).Info("Data container does not exist.")
+		log.WithField("name", do.Name).Info("Data container does not exist, creating it.")
 		ops := loaders.LoadDataDefinition(do.Name)
+		do.MakeDestination = true
```
Comment (ebuchman) [inline data/operate.go]: whats this for?

File: cmd/data.go [hunk 2]
```diff
@@ -178,6 +178,7 @@ func ImportData(cmd *cobra.Command, args []string) {
 	do.Name = args[0]
 	do.Source = args[1]
 	do.Destination = args[2]
+	do.MakeDestination = false // check if exists
```
Comment (ebuchman) [inline cmd/data.go]: dont we also need a flag?

File: data/operate.go [hunk 3]
```diff
@@ -46,7 +50,24 @@ func ImportData(do *definitions.Do) error {
...
+			}
+		} else {
+			doCheck := definitions.NowDo()
+			doCheck.Name = do.Name
+			doCheck.Operations.Args = []string{"ls", do.Destination}
+			_, err := ExecData(doCheck)
+			// if an error ls-ing, try importing again
+			// but by making the dir first
+			if err != nil {
+				do.MakeDestination = true
```
Comment (ebuchman) [inline data/operate.go]: whats the point of MakeDestination if we're just gonna go for it anyways?

Comment (zramsay) [inline data/operate.go]: it won't go for it here if there's no error listing...

**Comment by pietv** [comment]
LGTM :+1:

**Pull request merged**